.envrc

```diff
@@ -0,0 +1,6 @@
+if ! has nix_direnv_version || ! nix_direnv_version 2.2.0; then
+  source_url "https://raw.githubusercontent.com/nix-community/nix-direnv/2.2.0/direnvrc" "sha256-5EwyKnkJNQeXrRkYbwwRBcXbibosCJqyIUuz9Xq+LRc="
```

Comment:
are we pinning `direnv`'s version here?

Reply:
I don't actually remember where this came from :sweat_smile: 

I came back to the repo with a half-created direnv/flake env and went with it lol. I'll get rid of this bit.